src/main/java/worldofzuul/Game.java

```diff
@@ -145,13 +146,81 @@ private boolean processCommand(Command command) {
             interactPlayer();
         } else if (commandWord == commandWord.EXAMINE){
             examineObject(command);
-        }
-        else if (commandWord == CommandWord.HARVEST){
+        } else if (commandWord == CommandWord.HARVEST){
             harvestObject(command);
+        } else if (commandWord == CommandWord.SHOW){
+            showPlayerRoom(command);
         }
         return wantToQuit;
     }
 
+    //need to implement helper message on wrong 2'nd command
+    //need to fix collision when moving into objects
+    //DEPENDENCY: ROOMS MUST HAVE BEEN FILLED WITH STUFF, ELSE nullPointerExecption.
+    //BUG: move player east pass size of room for another room, then show local -> fix, make player always appear within room boundary when moving room to room.
+
+    private void showPlayerRoom(Command command){
+        if (!command.hasSecondWord()) {
+            MessageHelper.Command.unknownArgumentWhat(CommandWord.SHOW.toString());
+            return;
+        }
+        if("local".equals( command.getSecondWord() ) ){
+            for(int y=0; y < currentRoom.getRoomGrid().length; y++){
+                if(y>0){
+                    System.out.println();
+                }
+                for(int x=0; x < currentRoom.getRoomGrid()[0].length; x++){
+                    if((player.getPos().getX() == x ) && (player.getPos().getY() == y ) ){ //
+                        System.out.print("P  ");
+                    } else if (currentRoom.getRoomGrid()[y][x] instanceof Block) {
+                        System.out.print("[] ");
+                    } else if (currentRoom.getRoomGrid()[y][x] instanceof Field) {
+                        System.out.print("F  ");
+                    } else if (currentRoom.getRoomGrid()[y][x] instanceof Door) {
+                        System.out.print("D  "); //showGameWorld();
+                    }
+                }
+            }
+        } else if("global".equals(command.getSecondWord())) {
+//            showGlobal();
+              showGlobalStatic();
+
+        }
+        System.out.println(); //dont remove, makes room for userarrow
+    }
+
+    private void showGlobal(){
```

Comment:
For at få showGlobal til at virke hensigtsmæssige, kan det være at man skal ændre Room objektets Exit attribut til at være af typen Direction. Eftersom at Exit strenge ikke er fastsat til at være en korrekt retning og endvidere fremgår det ikke af Room at det behøver at være det.

Inden det bliver merget bør denne metode implementeres fuldstændigt eller revideres.